src/ContentScript/injectors/index.ts

```diff
@@ -8,7 +8,7 @@ class Injector {
       if (document.querySelector('.button-container')) {
         const ele = document.querySelector('.button-container') as HTMLElement;
         if (!ele.parentElement) return;
-        ele.parentElement.removeChild(ele);
+        // ele.parentElement.removeChild(ele);
```

Comment:
Please remove instead of commenting code out. If we want to revert later, we can always trace back the git history.